exercises/binary-search-tree/example.py

```diff
@@ -0,0 +1,69 @@
+class TreeNode(object):
+    def __init__(self, value):
+        self.value = value
+        self.left_node = None
+        self.right_node = None
+
+    def __str__(self):
+        return str(self.value)
+
+
+class BinarySearchTree(object):
+    def __init__(self):
+        self.root = None
+
+    def add(self, value):
+        if(self.root is None):
+            self.root = TreeNode(value)
+            inserted = True
+            return inserted
+        else:
+            inserted = False
+            cur_node = self.root
+
+            while not inserted:
+                if(value < cur_node.value):
+                    if(cur_node.left_node):
+                        cur_node = cur_node.left_node
+                    else:
+                        cur_node.left_node = TreeNode(value)
+                        inserted = True
+                        return inserted
+                elif(value > cur_node.value):
+                    if(cur_node.right_node):
+                        cur_node = cur_node.right_node
+                    else:
+                        cur_node.right_node = TreeNode(value)
+                        inserted = True
+                        return inserted
+
+    def search(self, searched_number):
```

Comment:
I recommend changing searched_number to either `number` or `search_number`, searched is in past tense which means you've already searched for it.

Reply:
Ok, changed!